client/my-sites/checkout/checkout/index.jsx

```diff
@@ -362,6 +363,10 @@ export class Checkout extends React.Component {
 			displayModeParam = { d: 'concierge' };
 		}
 
+		if ( hasTrafficGuide( cart ) ) {
```

Comment:
These functions are now unused. See also https://github.com/Automattic/wp-calypso/pull/48501 which will remove them.

Reply:
Thanks for the heads up! I have removed this code.